src/ducks/balance/components/AccountsList.jsx

```diff
@@ -17,7 +17,13 @@ class AccountsList extends React.PureComponent {
 
   goToTransactionsFilteredByDoc = account => () => {
     this.props.filterByDoc(account)
-    this.props.router.push('/transactions')
+
+    const isReimbursementsType = getAccountType(account) === 'Reimbursements'
+    const route = isReimbursementsType
+      ? '/balances/reimbursements'
+      : '/transactions'
+
+    this.props.router.push(route)
```

Comment:
you can export `filterByDoc` and `router` from `this.props` on the head of this function